src/common/hooks/useGetPlays.js

```diff
@@ -59,7 +59,9 @@ function useGetPlays() {
       res.forEach((element) => {
         element.title_name = toTitleCaseTrimmed(element.name);
       });
-      setPlays(res);
+      setPlays(
```

Comment:
It can be done through query object
For example.
```json
{
    "display": "Sorted play view",
    "name": "Fetch_Plays",
    "function": "plays",
    "write": false,
    "return": [
        "blog",
        "component",
        "cover",
        "created_at",
        "description",
        "featured",
        "github",
        "id",
        "language",
        {
            "level": [
                "name"
            ]
        },
        "name",
        "path",
        {
            "play_tags": {
                "tag": [
                    "name"
                ]
            }
        },
        "updated_at",
        {
            "user": [
                "id",
                "displayName",
                "avatarUrl"
            ]
        },
        "video"
    ],
    "orderBy": {
        "created_at": "asc"
    }
}
```

Reply:
I have updated